posthog/tasks/sync_all_organization_available_features.py

```diff
@@ -0,0 +1,9 @@
+from typing import Sequence, cast
+
+from posthog.models.organization import Organization
+
+
+def sync_all_organization_available_features() -> None:
```

Comment:
Nit: naming `sync_all_organizations_available_features`

Reply:
This is more of "Sync all `Organization.available_features`", I think I'll leave as is